ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__amd64-stable.yaml

```diff
@@ -524,14 +524,15 @@ tests:
     test:
     - chain: cucushift-installer-check-cluster-health
     workflow: cucushift-installer-rehearse-gcp-ipi-proxy-private
-- as: gcp-ipi-to-multiarch-f14
+- as: gcp-ipi-to-multiarch-f28
   cron: 4 11 3,19 * *
   steps:
     cluster_profile: gcp-qe
     env:
       ADDITIONAL_WORKER_VM_TYPE: t2a-standard-4
       ADDITIONAL_WORKERS: "1"
       COMPUTE_NODE_REPLICAS: "2"
+      ENABLE_OTA_TEST: OCP-60396
```

Comment:
I guess we also can run the testing on `azure-ipi-to-multiarch` and `aws-ipi-ovn-ipsec-to-multiarch`.

Reply:
i guess so. updated to use them, however worth noting `azure-ipi-to-multiarch` on amd64 is used by `OCP-53921` by Jia. now since we changed the frequency of testing, maybe worth adding OCP-53921 elsewhere as well?

Reply:
@jiajliu wdyt?

Reply:
> worth noting azure-ipi-to-multiarch on amd64 is used by OCP-53921 by Jia.

Got it.

Reply:
> now since we changed the frequency of testing, maybe worth adding OCP-53921 elsewhere as well?

case ocp-53921 does not require a high frequency, so I'm fine with `f14->f28`.

Reply:
sounds good! if no other issues, I'll go ahead and backport.

Reply:
to keep consistent, maybe we can enable OCP-53921 on the same job in arm ci?

Reply:
I see 60397 is added on the same profile in arm job now, and 53921 is only for amd64 currently. so i think that's fine to host different e2e case on dedicated profile for different test purpose.

Reply:
so `OCP-53921` only works with amd64, right?

Reply:
53921 could be tested through arm64-heter or amd64-heter(but it does not matter which one for the checkpoint itself), so when implementing the case in prow ci, we pick one of amd64 profile only for:
1. it does not matter profile or arch
2. one job could make checking test result easily

Reply:
Got it, on your call.

Reply:
if just for consistent, i think we could keep it on amd64 only. but if we see any risk to cover it on amd64 only, i'm fine to file an extra pr to enable it later.

Reply:
honestly, for my test cases I as well don't really need it to be tested on more than one profile on each case. currently I"m backporting it enabled in all the different profiles (as Jianlin suggested), but I'm fine with just one profile per test case and leave the rest as just a regular CI profiles

Reply:
On your call as the same. A advantage of enabling the case on multiple cases is you are easier to collect test results, not have to wait the only result in one month.